ui/src/dashboards/components/GraphOptionsCustomizableColumn.js

```diff
@@ -8,15 +8,16 @@ const GraphOptionsCustomizableColumn = ({
   onColumnRename,
 }) => {
   return (
-    <div className="gauge-controls--section">
-      <div className="gauge-controls--label">
+    <div className="column-controls--section">
+      <div className="column-controls--label">
         {originalColumnName}
       </div>
       <InputClickToEdit
         value={newColumnName}
-        wrapperClass="fancytable--td orgs-table--name"
+        wrapperClass="column-controls-input"
         onUpdate={onColumnRename}
         placeholder="Rename..."
+        appearAsNormalInput={true}
```

Comment:
hahahah. I feel like we might all have an `appearAsNormal` property that we can turn on and off.

Reply:
😺